src/ConsoleBuffer/Commands/SetMode.cs

```diff
@@ -27,12 +27,12 @@ public SetMode(string bufferData, bool set) : base(bufferData)
 
             if (this.Parameters.Count == 1)
             {
-                switch (this.Parameters[0])
+                switch (this.Parameters.GetValue(0))
```

Comment:
It probably makes sense to just add an indexer to the ParsedParameters type here instead of adding a GetValue(n) method?

Reply:
If I add an indexer, it makes the code a bit cleaner, but then pushes the responsibility of checking for errors and using a default value to the callers.  In the end, not a bad thing in my opinion, since it localizes the error case/default value to the caller, where it matters. Your call.